src/core/client/ui/encapsulation/useWindowBreakpointClasses.ts

```diff
@@ -26,7 +26,9 @@ export default function useWindowBreakpointClasses() {
     };
     window.addEventListener("resize", callback);
     return () => {
-      window.removeEventListener("resize", callback);
+      if (window.removeEventListener) {
```

Comment:
If `window` or `removeEventListener` were not defined, that would set alarm bells in my head that something is weird for the context we're running in.

Is this a web worker or service worker issue in Electron or PWA code running Coral?

Reply:
Hmmm. This is definitely strange - agreed that this fix addresses the errors, but doesn't really explain why `window` is undefined and will likely be resulting in listener leaks. Weird one.

The PWA environment isn't executing Coral within a web worker or service worker (there is a service worker, but that's operating in prepopulated-cache-or-back-to-origin mode, and shouldn't touch these scripts). It definitely shouldn't mess with anything like `window` 😅